### ISSUE TITLE
Document Persistence into the database

### ISSUE THREAD

**Issue by mtangoo** [moneyphp/money#454]
Hi,
Thanks for the Library.
I have read documentation but persisting data is missing. There a way too many suggestions as to how one should persist money in the database that requires the library to document recommended way(s) to store Money object in the database.

To add to confusion, databases like Postgres have money fields. It would be great to touch on such subject so that we might know if we can benefit with those data types when using this library.

TL;DR - I believe there is a need to say something about database persistence of objects

**Comment by frederikbosch** [comment]
Sorry. We don’t take the responsibility for persistency. You might look at previous issues for other people’s solutions. Good luck!

**Comment by sagikazarmark** [comment]
There were discussions in the past, but those were mostly related to concrete DBAL layers (doctrine, laravel, etc).

Not sure if we can spend time on doing some experiments right now, but we certainly happy to review PRs and provide a place for this information.

**Comment by mtangoo** [comment]
>Sorry. We don’t take the responsibility for persistency. 

Oh! Sorry for bothering.

>You might look at previous issues for other people’s solutions. Good luck!

That was the first thing to do. 

Thanks for responding anyway!
